vassal-app/src/main/java/VASSAL/configure/ListKeyCommandsDialog.java

```diff
@@ -202,13 +240,38 @@ public void actionPerformed(ActionEvent e) {
     }
   }
 
+  public class JumpAction extends AbstractAction {
+    private static final long serialVersionUID = 1L;
+    private final JTable table;
+
+    public JumpAction(JTable table) {
+      this.table = table;
+      putValue(NAME, Resources.getString("Editor.ListKeyCommands.jump_to_definition"));
+    }
+
+    @Override
+    public void actionPerformed(ActionEvent e) {
+      int rowIndex = table.getSelectedRow();
+      if (rowIndex < 0) return;
+
+      rowIndex = table.convertRowIndexToModel(rowIndex);
+
+      final AbstractConfigurable jumpTo = ((MyTableModel)(table.getModel())).getSourceFor(rowIndex);
+
+      if (jumpTo instanceof AbstractConfigurable) {
```

Comment:
`jumpTo` has to be an `AbstractConfigurable` because you just declared it to be one.

Reply:
Oh right there were various earlier versions where it wasn't.